central/complianceoperator/v2/remediations/datastore/datastore_impl.go

```diff
@@ -0,0 +1,40 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/central/complianceoperator/v2/remediations/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/search"
+)
+
+var _ DataStore = (*datastoreImpl)(nil)
```

Comment:
What is this for?

Reply:
It asserts that the interface is implemented at compile time. If it is not implemented it crashes. Additionally IntelliJ is able to check this too during development.